Cake.Recipe/Content/build.cake

```diff
@@ -398,35 +398,95 @@ public void CopyBuildOutput(BuildVersion buildVersion)
         if (parsedProject.IsLibrary() && (parsedProject.HasPackage("xunit") || parsedProject.HasReference("xunit.core")))
         {
             Information("Project has an output type of library and is an xUnit Test Project: {0}", parsedProject.RootNameSpace);
-            var outputFolder = BuildParameters.Paths.Directories.PublishedxUnitTests.Combine(parsedProject.RootNameSpace);
-            EnsureDirectoryExists(outputFolder);
-            CopyFiles(GetFiles(parsedProject.OutputPath.FullPath + "/**/*"), outputFolder, true);
-            continue;
+
+            if (parsedProject.IsVS2017ProjectFormat)
+            {
+                foreach (var outputPath in parsedProject.OutputPaths)
+                {
+                    var outputFolder = BuildParameters.Paths.Directories.PublishedxUnitTests.Combine(parsedProject.RootNameSpace).Combine(outputPath.GetDirectoryName());
+                    EnsureDirectoryExists(outputFolder);
+                    CopyFiles(GetFiles(outputPath + "/**/*"), outputFolder, true);
+                }
+
+                continue;
+            }
+            else
+            {
+                var outputFolder = BuildParameters.Paths.Directories.PublishedxUnitTests.Combine(parsedProject.RootNameSpace);
+                EnsureDirectoryExists(outputFolder);
+                CopyFiles(GetFiles(parsedProject.OutputPaths[0].FullPath + "/**/*"), outputFolder, true);
+                continue;
+            }
         }
         else if (parsedProject.IsLibrary() && (parsedProject.HasPackage("fixie") || parsedProject.HasReference("fixie")))
         {
             Information("Project has an output type of library and is a Fixie Project: {0}", parsedProject.RootNameSpace);
-            var outputFolder = BuildParameters.Paths.Directories.PublishedFixieTests.Combine(parsedProject.RootNameSpace);
-            EnsureDirectoryExists(outputFolder);
-            CopyFiles(GetFiles(parsedProject.OutputPath.FullPath + "/**/*"), outputFolder, true);
-            continue;
+
+            if (parsedProject.IsVS2017ProjectFormat)
```

Comment:
We may perhaps think about refactoring this whole if/else statements to a helper method, instead of duplicating the effort multiple times.

Reply:
Same comment as above..